recipes/viral_consensus/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.0.4" %}
+
+package:
+  name: viral_consensus
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('viral_consensus', max_pin="x.x.x") }}
+
+source:
+  url: https://github.com/niemasd/ViralConsensus/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d531311c8e5c6b689f4426a9e89cc00426733ae9d4ff71757a861ad6ddbf81f4 # curl -sL URL | openssl sha256
```

Comment:
In future the autobump script will show you new sha256 values, so you won't need the mnemonic comment.

Reply:
Thanks! I more so left it as a note for myself if I ever add more tools to bioconda in the future, but I'll remove it for cleanliness (it was easy to Google anyways)